chain-tx-enclave-next/mls/src/key.rs

```diff
@@ -100,6 +104,14 @@ impl HPKEPrivateKey {
     pub fn kex_secret(&self) -> &<hpke::kex::DhP256 as hpke::KeyExchange>::PrivateKey {
         &self.0
     }
+
+    pub fn unmarshal(secret: &[u8]) -> Result<Self, HpkeError> {
```

Comment:
consider `decode` for unmarshal, because we're using `encode`, `decode` terms

Reply:
these are keywords used in the spec